Source/ChilliSource/Rendering/Base/ForwardRenderPassCompiler.cpp

```diff
@@ -481,33 +488,41 @@ namespace ChilliSource
     }
     
     //------------------------------------------------------------------------------
-    std::vector<TargetRenderPassGroup> ForwardRenderPassCompiler::CompileTargetRenderPassGroups(const TaskContext& taskContext, const RenderFrame& renderFrame) noexcept
+    std::vector<TargetRenderPassGroup> ForwardRenderPassCompiler::CompileTargetRenderPassGroups(const TaskContext& taskContext, std::vector<RenderFrame>&& renderFrames) noexcept
     {
-        auto numTargets = CalcNumTargets(renderFrame);
+        u32 numTargets = 0;
+        for(const auto& renderFrame : renderFrames)
+        {
+            numTargets += CalcNumTargets(renderFrame);
+        }
+        
         std::vector<TargetRenderPassGroup> targetRenderPassGroups(numTargets);
         std::vector<Task> tasks;
         u32 nextPassIndex = 0;
         
-        // Shadow targets
-        for (const auto& directionalRenderLight : renderFrame.GetDirectionalRenderLights())
+        for(const auto& renderFrame : renderFrames)
```

Comment:
A RenderFrame was called a "Frame" because it contained everything a single frame required to render. If a frame is now comprised of multiple RenderFrames then this no longer makes sense.

You should either make render frame comprise of a series sub frame object that describe the RTT portion of the frame, or rename RenderFrame to something that is more representative of it's new purpose.